spec/cordial/products_spec.rb

```diff
@@ -46,8 +46,10 @@
     subject do
       described_class.create(id: product_id,
                              name: product_name,
-                             price: price,
-                             variants: variants)
+                             options: {
+                               price: price,
+                               variants: variants
+                             })
```

Comment:
Can we do another test with a more complex options hash? I wonder if we need to call `.to_json`

Reply:
I have updated the test adding more optional attributes :) and re-recording the VCR and it's working fine 😄